west.yml

```diff
@@ -71,7 +71,7 @@ manifest:
       revision: ff9b7f295da7e8918fbe3e0119b5271cb9cb4a55
       path: modules/hal/stm32
     - name: hal_ti
-      revision: 811363a384cbd2d2b699cae7b8c4375de8218b65
+      revision: ebf0a6f11cd6ff9f735db8391a6351ffeeda4cc3
```

Comment:
Shouldn't this be in a separate commit?   Assuming there are HAL dependencies required for this (I don't see any...).

Reply:
Yes, this would be cleaner. I will fix this too.

Reply:
Actually, no. When I remove the legacy timeouts option from the `config SOC_FAMILY_TISIMPLELINK` every single sample will start to fail to build if the HAL itself is not updated. I will tweak the commits a bit, but this needs to be together at least with the removal of the Kconfig select.

Reply:
I don't really see that; a west update commit could precede the removal of the default to use the old API, unless the HAL breaks with legacy timeout (which would not be a great solution path).  Not a blocker, the change is now explained.